driver/src/main/java/org/neo4j/driver/internal/metrics/ListenerEvent.java

```diff
@@ -20,7 +20,7 @@
 
 public interface ListenerEvent
 {
-    void start();
-    long elapsed();
+    void start( long currentTime );
```

Comment:
Wdyt if we leave the interface as earlier but pass the `Clock` instance to the `TimeRecorderListenerEvent`?